src/absil/il.fs

```diff
@@ -165,7 +165,7 @@ let splitTypeNameRight nm =
 
 /// This is used to store event, property and field maps.
 ///
-/// Review: this is not such a great data structure.
+/// REVIEW: this is not such a great data structure.
```

Comment:
This is not an actionable review comment.  With no detail why it is "not such a great data structure"  I would say lose the comment.  In principle, Review: should probably point out a future course of action.